xcell/cls/cl.py

```diff
@@ -251,17 +294,23 @@ def compute_tracer_ccl(self, tr, mapper):
         dtype = mapper.get_dtype()
         tracer = self.data.data['tracers'][tr]
         fiducial = self.data.data['cov']['fiducial']
+        ccl_pr = self.hm_par['prof_NFW']
+        ccl_pr_2pt = None
```

Comment:
Is `ccl_pr_2pt` always None? If so, why is it interesting to return it? If not, I'd add it as an option to be read from the file.

Reply:
No, check out the HOD case

Reply:
Oh, I see! Missed that line. Sorry!